src/libcore/macros.rs

```diff
@@ -8,6 +8,16 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+#[macro_export]
+// This stability attribute is totally useless.
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
The attribute is useless because a) stability attributes on macros are generally useless, and b) because of the `#[cfg(stage0)]` below. This won't have any impact on anything presented to users of the compiler. This entire macro definition may be removed if we switch stage0.